File: Selenium/Activities/Activity4.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.firefox import GeckoDriverManager

service = FirefoxService(GeckoDriverManager().install())

# Start the Driver
with webdriver.Firefox(service=service) as driver:
    driver.get("https://v1.training-support.net/selenium/target-practice")
   
    print("Home title is: ", driver.title)

    thirheader = driver.find_element(By.XPATH,"//h3[@class='ui yellow header']")
    print("Third Header: ",thirheader.text)
    fifthheader=driver.find_element(By.XPATH,"//h5[@class='ui green header']").value_of_css_property('color')
    print("Fifth Header: ",fifthheader)
    violet_button = driver.find_element(By.CLASS_NAME, "violet")
    print("Violet button's classes are: ", violet_button.get_attribute("class"))

    greytext = driver.find_element(By.CLASS_NAME,"grey")
    print("Grey text: ", greytext.text)

    driver.close()

File: Selenium/Activities/Activity4.java
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity4 {
    public static void main(String[] args) {

        WebDriver driver = new FirefoxDriver();
        driver.get("https://v1.training-support.net/selenium/target-practice");
        System.out.println(driver.getTitle());

        String thirdHeader = driver.findElement(By.xpath("//h3[@class='ui yellow header']")).getText();
        System.out.println("Third Header: " + thirdHeader);

        String fifthheader = driver.findElement(By.xpath("//h5[@class='ui green header']")).getCssValue("color");
        System.out.println("Fifth Header: " + fifthheader);

        String violet_button = driver.findElement(By.className("violet")).getAttribute("class");
        System.out.println("Violet button's classes are: " + violet_button);

        String greytext = driver.findElement(By.className("grey")).getText();
        System.out.println("Grey text: " + greytext);
        driver.close();
    }}